_data/Fernisty/Fernisty.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Lily # No longer than 28 characters
+institution: University of Nottingham 🚩 # no longer than 58 characters
+profile_pic: fern.png # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
```

Comment:
Incorrect file name, please do one of the following:
1. Change `fern.png` to `Fernisty.png` in the markdown file.
2. Rename `Fernisty.png` to `fern.png`.
I'd suggest you make the change in the markdown file!